### PR TITLE
chore: Support offsite message

### PR DESCRIPTION
**Pull request by joethreepwood**
## Problem

DO NOT MERGE

We have an offsite, followed by OOO from our US team, planned. As a result, we may want to set better expectation about support and reassure users that 1) we're still here 2) we're improving things. 

This is a first crack at the message, which will appear in place of the usual response time estimates. Very open to feedback on the copy, as well as ways to better integrate it into the sidebar. 

One thing that may be very helpful here @slshults is getting Max AI shipped before the offsite, so we can point to him as a stand-in. How likely do you feel that is?

## How did you test this code?

Untested.

### CODE REVIEW

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 1]
```diff
@@ -126,33 +126,12 @@ const SupportFormBlock = ({ onCancel }: { onCancel: () => void }): JSX.Element =
             <div className="grid grid-cols-2 border rounded [&_>*]:px-2 [&_>*]:py-0.5 mb-4 bg-bg-light pt-4">
                 <div className="col-span-full flex justify-between border-b bg-bg-white py-1">
                     <div>
-                        <strong>Avg support response times</strong>
+                        <strong>We're making improvements:</strong> Our team is attending an offsite so we can make long-term enhancements. We're working slightly different hours, so non-urgent inquiries without priority support may experience a slight delay. We’ll be back to full speed soon!
```
Comment (charlescook-ph) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:129]:
```suggestion
                        <strong>We're making improvements:</strong> Our team is attending an offsite so we can make long-term enhancements. We're working different hours, so non-urgent inquiries without priority support may experience a slight delay. We’ll be back to full speed next week!
```

Comment (charlescook-ph) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:129]: Suggested being specific about when we're back, and also removed duplicate 'slight'

**Comment by slshults** [comment]
Alas @joethreepwood, getting Max shipped to users before the offsite is unlikely, particular with ticket volumes up atm. I'll need internal testing by hoglets (behind a feature flag) to work on the rate-limiting, concurrent chats etc. We'll also need to do a phased rollout because Anthropic only allows the raising of [rate-limits](https://docs.anthropic.com/en/api/rate-limits) after usage shows it's warranted: "- Limits are defined by usage tier, where each tier is associated with a different set of spend and rate limits. - Your organization will increase tiers automatically as you reach certain thresholds while using the API."

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 2]
```diff
@@ -120,33 +120,12 @@ const SupportFormBlock = ({ onCancel }: { onCancel: () => void }): JSX.Element =
...
                 <div className="col-span-full flex justify-between border-b bg-bg-white py-1">
                     <div>
-                        <strong>Avg support response times</strong>
+                        <strong>We're making improvements:</strong> Our team is attending an offsite so we can make long-term enhancements. We're working different hours, so non-urgent inquiries without priority support may experience a slight delay. We’ll be back to full speed next week!
                     </div>
                     <div>
                         <Link to={urls.organizationBilling([ProductKey.PLATFORM_AND_SUPPORT])}>Explore options</Link>
                     </div>
                 </div>
-                {supportPlans?.map((plan) => {
```
Comment (zlwaterfield) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:129]: Can we just comment this out for now?

**Review by zlwaterfield — APPROVED** [review]
Message looks good, but one request to comment out instead of removing existing table.

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 3]
```diff
@@ -120,33 +120,12 @@ const SupportFormBlock = ({ onCancel }: { onCancel: () => void }): JSX.Element =
             <div className="grid grid-cols-2 border rounded [&_>*]:px-2 [&_>*]:py-0.5 mb-4 bg-bg-light pt-4">
                 <div className="col-span-full flex justify-between border-b bg-bg-white py-1">
                     <div>
-                        <strong>Avg support response times</strong>
+                        <strong>We're making improvements:</strong> Our team is attending an offsite so we can make long-term enhancements. We're working different hours, so non-urgent inquiries without priority support may experience a slight delay. We’ll be back to full speed next week!
                     </div>
                     <div>
                         <Link to={urls.organizationBilling([ProductKey.PLATFORM_AND_SUPPORT])}>Explore options</Link>
                     </div>
                 </div>
```
Comment (joethreepwood) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:128]:
```suggestion
                </div>
                // {supportPlans?.map((plan) => {

```

Comment (joethreepwood) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:128]: This should comment this out for you, @zlwaterfield

Comment (zlwaterfield) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:128]: That doesn't look right, ill commit a change for it.

**Pull request merged**